src/apps/leaderboards/models.py

```diff
@@ -5,19 +5,23 @@
 class Leaderboard(models.Model):
     ADD = "Add"
     ADD_DELETE = "Add_And_Delete"
+    ADD_DELETE_MULTIPLE = "Add_And_Delete_Multiple"
     ADD_BY_REVIEW = "Add_By_Review"
     FORCE_LAST = "Force_Last"
     FORCE_LATEST_MULTIPLE = "Force_Latest_Multiple"
     FORCE_BEST = "Force_Best"
     SUBMISSION_RULES = (
         (ADD, "Only allow adding one submission"),
         (ADD_DELETE, "Allow users to add a single submission and remove that submission"),
+        (ADD_DELETE_MULTIPLE, "Allow users to add a multiple submissions and remove those submission"),
         (ADD_BY_REVIEW, "Only allow adding one submission that is approved by review"),
         (FORCE_LAST, "Force only the last submission"),
         (FORCE_LATEST_MULTIPLE, "Force latest submission to be added to leaderboard (multiple)"),
         (FORCE_BEST, 'Force only the best submission to the leaderboard')
     )
 
+    AUTO_SUBMISSION_RULES = [FORCE_LAST, FORCE_BEST, FORCE_LATEST_MULTIPLE]
+
     primary_index = models.PositiveIntegerField(default=0)
     title = models.CharField(max_length=64)
     key = models.CharField(max_length=36)
```

Comment:
Should we set a default submission rule here rather than `null`?

Reply:
We could default to "Add" and set it `null=False`?